include/dem/insertion.h

```diff
@@ -159,6 +164,31 @@ class Insertion
     const DEMSolverParameters<dim> &dem_parameters,
     const ConditionalOStream       &pcout);
 
+  /**
+   * @brief Find every cell that are completely and partially inside de the
+   * clearing box. For the cell completely inside the box, all the particles
+   * will be deleted. For those partially inside, a fine verification is
+   * required.
+   *
+   * @param triangulation Triangulation to access the cells in which the
+   * particles deleted.
+   */
+  void
+  find_in_clearing_box_cells(
+    const parallel::distributed::Triangulation<dim> &triangulation);
+
+
+  /**
+   * @brief Remove every particle located inside a predefined zone. Currently,
+   * the zone is defined by a box.
+   *
+   * @param particle_handler The particle handler of particles which are being
+   * removed.
+   */
+  void
+  clear_particle_box(Particles::ParticleHandler<dim> &particle_handler);
```

Comment:
remove_particles_in_box

or something like that. What is particle_box. This is a bit indefinite.